src/browser-frontend/views/browser/chrome/tabbar/tab.css

```diff
@@ -47,6 +48,32 @@ specific language governing permissions and limitations under the License.
   height: var(--theme-tab-open-height);
 }
 
+.tab.has-owner {
+  left: 0;
+  transition:
+    width 0.05s var(--animation-curve),
+    background-position 0.2s var(--animation-curve),
+    height 0.2s var(--animation-curve),
+    left 0.2s var(--animation-curve);
```

Comment:
Nit: order these properties to follow "idiomatic css": left, width, height, background-position.